UK/vATIS/ADC/Cranfield(EGTC).json

```diff
@@ -118,19 +118,24 @@
                 },
                 "transitionLevel": {
                     "values": [
+                        {
+                            "low": 1050,
+                            "high": 1060,
+                            "altitude": 60
+                        },
                         {
                             "low": 1032,
                             "high": 1049,
                             "altitude": 65
                         },
                         {
-                            "low": 1014,
+                            "low": 1013,
```

Comment:
It's unclear if this is true except ukcp datasource.
https://ukcp.vatsim.uk/airfields/53